admin/src/templates/header.html

```diff
@@ -4,9 +4,27 @@
       <h1 translate>admin.app.name</h1>
     </div>
     <div class="extras">
-      <a href="#" ng-click="logout()">
-        <span translate>Log Out</span>
-      </a>
+      <span class="dropdown options">
+        <a data-toggle="dropdown">
+          <div class="mm-icon mm-icon-inverse">
+            <span class="fa fa-fw fa-bars"></span>
+          </div>
+        </a>
+        <ul role="menu" class="dropdown-menu mm-dropdown-menu">
+          <li role="presentation" mm-auth="can_configure">
+            <a id="app" ng-href="{{baseUrl}}" role="menuitem" tabindex="-1" rel="noopener noreferrer">
+              <i class="fa fa-fw fa-cog"></i>
```

Comment:
This is not the right icon for the webapp - have a look through the [icon library](https://fontawesome.com/v4.7.0/icons/) and see if anything else makes sense.